### PR TITLE
distrobox-*: two-lined descriptions

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
The change LGTM, while we do prefer having 2 lines of description + 1 line of more information link, we do allow exceptions in pages like Pacman so IG it is fine (I initially went with 2 lines to not cross this limit).

**Review by kbdharun — COMMENTED** [review]
I don't think we need to specify " Subcommand of `distrobox`." on every page, the see also reference would be enough. Other than that the changes LGTM.

If possible can you enclose distrobox in descriptions inside backticks?

File: pages/linux/distrobox-host-exec.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # distrobox-host-exec
 
-> Execute a command on the host from inside a distrobox container.
-> Subcommand of `distrobox`. See also: `tldr distrobox`.
+> Execute a command on the host from inside a distrobox container. See also: `tldr distrobox`.
```
Comment (kbdharun) [inline pages/linux/distrobox-host-exec.md:3]:
```suggestion
> Execute a command on the host from inside a `distrobox` container. See also: `tldr distrobox`.
```
By enclosing it in backticks, I meant this.

File: pages.pt_BR/linux/distrobox-rm.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 > Subcomando de `distrobox`. Veja também: `tldr distrobox`.
 > Mais informações: <https://distrobox.it/usage/distrobox-rm>.
 
-- Remover um contêiner distrobox (Dica: Pare o contêiner antes de removê-lo):
+- Remover um contêiner Distribox (Dica: Pare o contêiner antes de removê-lo):
 
 `distrobox-rm {{nome_do_contêiner}}`
 
-- Remover um contêiner distrobox forçadamente:
+- Remover um contêiner Distribox forçadamente:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-rm.md:11]:
```suggestion
- Remover um contêiner Distrobox forçadamente:
```

File: pages.pt_BR/linux/distrobox-rm.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # distrobox-rm
 
-> Remover um contêiner distrobox.
+> Remover um contêiner Distribox.
 > Subcomando de `distrobox`. Veja também: `tldr distrobox`.
 > Mais informações: <https://distrobox.it/usage/distrobox-rm>.
 
-- Remover um contêiner distrobox (Dica: Pare o contêiner antes de removê-lo):
+- Remover um contêiner Distribox (Dica: Pare o contêiner antes de removê-lo):
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-rm.md:7]:
```suggestion
- Remover um contêiner Distrobox (Dica: Pare o contêiner antes de removê-lo):
```

File: pages.pt_BR/linux/distrobox-rm.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 # distrobox-rm
 
-> Remover um contêiner distrobox.
+> Remover um contêiner Distribox.
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-rm.md:3]:
```suggestion
> Remover um contêiner Distrobox.
```

File: pages.pt_BR/linux/distrobox-host-exec.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
 # distrobox-host-exec
 
-> Executar um comando no sistema host a partir do interior de um contêiner distrobox.
+> Executar um comando no sistema host a partir do interior de um contêiner Distribox.
 > Subcomando de `distrobox`. Veja também: `tldr distrobox`.
 > Mais informações: <https://distrobox.it/usage/distrobox-host-exec>.
 
-- Executar um comando no sistema host a partir do interior do contêiner distrobox:
+- Executar um comando no sistema host a partir do interior do contêiner Distribox:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-host-exec.md:7]:
```suggestion
- Executar um comando no sistema host a partir do interior do contêiner Distrobox:
```

File: pages.pt_BR/linux/distrobox-enter.md [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
 
 `distrobox-enter {{nome_do_contêiner}}`
 
-- Entrar em um contêiner distrobox e executar um comando no login:
+- Entrar em um contêiner Distribox e executar um comando no login:
 
 `distrobox-enter {{nome_do_contêiner}} -- {{sh -l}}`
 
-- Entrar em um contêiner distrobox sem instanciar um tty:
+- Entrar em um contêiner Distribox sem instanciar um tty:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-enter.md:16]:
```suggestion
- Entrar em um contêiner Distrobox sem instanciar um tty:
```

File: pages.pt_BR/linux/distrobox-enter.md [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
 > Se usado dentro de um script, um aplicativo ou um serviço, você pode usar o modo `--headless` para desabilitar o tty e a interatividade.
 > Mais informações: <https://distrobox.it/usage/distrobox-enter>.
 
-- Entrar em um contêiner distrobox:
+- Entrar em um contêiner Distribox:
 
 `distrobox-enter {{nome_do_contêiner}}`
 
-- Entrar em um contêiner distrobox e executar um comando no login:
+- Entrar em um contêiner Distribox e executar um comando no login:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-enter.md:12]:
```suggestion
- Entrar em um contêiner Distrobox e executar um comando no login:
```

File: pages.pt_BR/linux/distrobox-enter.md [hunk 8]
```diff
@@ -1,18 +1,18 @@
 # distrobox-enter
 
-> Entrar em um contêiner distrobox. Veja também: `tldr distrobox`.
+> Entrar em um contêiner Distribox. Veja também: `tldr distrobox`.
 > O comando padrão executado é o seu SHELL, mas você pode especificar shells diferentes ou comandos completos para serem executados.
 > Se usado dentro de um script, um aplicativo ou um serviço, você pode usar o modo `--headless` para desabilitar o tty e a interatividade.
 > Mais informações: <https://distrobox.it/usage/distrobox-enter>.
 
-- Entrar em um contêiner distrobox:
+- Entrar em um contêiner Distribox:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-enter.md:8]:
```suggestion
- Entrar em um contêiner Distrobox:
```

File: pages.pt_BR/linux/distrobox-enter.md [hunk 9]
```diff
@@ -1,18 +1,18 @@
 # distrobox-enter
 
-> Entrar em um contêiner distrobox. Veja também: `tldr distrobox`.
+> Entrar em um contêiner Distribox. Veja também: `tldr distrobox`.
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-enter.md:3]:
```suggestion
> Entrar em um contêiner Distrobox. Veja também: `tldr distrobox`.
```

File: pages.pt_BR/linux/distrobox-create.md [hunk 10]
```diff
@@ -1,13 +1,13 @@
...
 > O contêiner criado será integrado ao sistema host, permitindo o compartilhamento do diretório HOME do usuário, armazenamento externo, dispositivos USB externos, aplicativos gráficos (X11/Wayland) e áudio.
 > Mais informações: <https://distrobox.it/usage/distrobox-create>.
 
-- Criar um contêiner distrobox usando a imagem do Ubuntu:
+- Criar um contêiner Distribox usando a imagem do Ubuntu:
 
 `distrobox-create {{nome_do_contêiner}} --image {{ubuntu:latest}}`
 
-- Clonar um contêiner distrobox:
+- Clonar um contêiner Distribox:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-create.md:11]:
```suggestion
- Clonar um contêiner Distrobox:
```

File: pages.pt_BR/linux/distrobox-create.md [hunk 11]
```diff
@@ -1,13 +1,13 @@
 # distrobox-create
 
-> Criar um contêiner distrobox. Veja também: `tldr distrobox`.
+> Criar um contêiner Distribox. Veja também: `tldr distrobox`.
 > O contêiner criado será integrado ao sistema host, permitindo o compartilhamento do diretório HOME do usuário, armazenamento externo, dispositivos USB externos, aplicativos gráficos (X11/Wayland) e áudio.
 > Mais informações: <https://distrobox.it/usage/distrobox-create>.
 
-- Criar um contêiner distrobox usando a imagem do Ubuntu:
+- Criar um contêiner Distribox usando a imagem do Ubuntu:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-create.md:7]:
```suggestion
- Criar um contêiner Distrobox usando a imagem do Ubuntu:
```

File: pages.pt_BR/linux/distrobox-create.md [hunk 12]
```diff
@@ -1,13 +1,13 @@
 # distrobox-create
 
-> Criar um contêiner distrobox. Veja também: `tldr distrobox`.
+> Criar um contêiner Distribox. Veja também: `tldr distrobox`.
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-create.md:3]:
```suggestion
> Criar um contêiner Distrobox. Veja também: `tldr distrobox`.
```

File: pages.pt_BR/linux/distrobox-host-exec.md [hunk 13]
```diff
@@ -1,10 +1,10 @@
 # distrobox-host-exec
 
-> Executar um comando no sistema host a partir do interior de um contêiner distrobox.
+> Executar um comando no sistema host a partir do interior de um contêiner Distribox.
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-host-exec.md:3]:
```suggestion
> Executar um comando no sistema host a partir do interior de um contêiner Distrobox.
```

File: pages.pt_BR/linux/distrobox-stop.md [hunk 14]
```diff
@@ -1,13 +1,13 @@
 # distrobox-stop
 
-> Parar um contêiner distrobox.
+> Parar um contêiner Distribox.
 > Subcomando de `distrobox`. Veja também: `tldr distrobox`.
 > Mais informações: <https://distrobox.it/usage/distrobox-stop>.
 
-- Parar um contêiner distrobox:
+- Parar um contêiner Distribox:
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-stop.md:7]:
```suggestion
- Parar um contêiner Distrobox:
```

File: pages.pt_BR/linux/distrobox-stop.md [hunk 15]
```diff
@@ -1,13 +1,13 @@
 # distrobox-stop
 
-> Parar um contêiner distrobox.
+> Parar um contêiner Distribox.
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-stop.md:3]:
```suggestion
> Parar um contêiner Distrobox.
```

File: pages.pt_BR/linux/distrobox-stop.md [hunk 16]
```diff
@@ -1,13 +1,13 @@
...
 > Subcomando de `distrobox`. Veja também: `tldr distrobox`.
 > Mais informações: <https://distrobox.it/usage/distrobox-stop>.
 
-- Parar um contêiner distrobox:
+- Parar um contêiner Distribox:
 
 `distrobox-stop {{nome_do_contêiner}}`
 
-- Parar um contêiner distrobox de forma não interativa (sem confirmação):
+- Parar um contêiner Distribox de forma não interativa (sem confirmação):
```
Comment (isaacvicente) [inline pages.pt_BR/linux/distrobox-stop.md:11]:
```suggestion
- Parar um contêiner Distrobox de forma não interativa (sem confirmação):
```

**Review by isaacvicente — COMMENTED** [review]
I think you've made a typo; instead of Distrobox, it was Distribox.
I will commit my own suggestions for everyone's time sake.

Thank you very much for your contribution!! We really appreciate it.

File: pages/linux/distrobox.md [hunk 17]
```diff
@@ -1,7 +1,6 @@
 # distrobox
 
-> Use any Linux distribution inside your terminal in a container. Install & use packages inside it while tightly integrating with the host OS, sharing storage (`home` directory) and hardware.
-> It uses Podman or Docker to create your containers.
+> Use any Linux distribution inside your terminal in a container. Install & use packages inside it while tightly integrating with the host OS, sharing storage (`home` directory) and hardware. It uses Podman or Docker to create your containers.
```
Comment (kbdharun) [inline pages/linux/distrobox.md:3]:
```suggestion
> Use any Linux distribution inside your terminal in a container. Install & use packages inside it while tightly integrating with the host OS, sharing storage (`home` directory) and hardware.
> Note: It uses Podman or Docker to create your containers.
```

**Review by kbdharun — APPROVED** [review]

**Review by leonvsc — APPROVED** [review]
LGTM!

**Pull request merged**